astropy/coordinates/angles.py

```diff
@@ -632,10 +632,10 @@ def _wrap_internal(self):
         wrap_angle_floor = wrap_angle - a360
         self_angle = self.value
         # Do the wrapping, but only if any angles need to be wrapped
-        if np.any(self_angle < wrap_angle_floor) or np.any(self_angle >= wrap_angle):
-            wrapped = np.mod(self_angle - wrap_angle, a360) + wrap_angle_floor
-            value = u.Quantity(wrapped, self.unit)
-            super().__setitem__((), value)
+        with np.errstate(invalid='ignore'):
```

Comment:
Huh? Do you still need this? That's a pity, though definitely the right way to go if it is needed, so approval remains.

Reply:
Yes, a couple of the tests were failing without it (see https://travis-ci.org/astropy/astropy/builds/617419717?utm_source=github_status&utm_medium=notification for the failing tests before I force pushed the branch)

Reply:
OK, thanks. It seems there is a numpy version dependence. Anyway, now it is all good, so will merge. Thanks!